src/compute/src/sink/subscribe.rs

```diff
@@ -86,22 +86,24 @@ fn subscribe<G>(
 {
     let mut results = Vec::new();
     let mut finished = false;
+    let mut rows = Default::default();
```

Comment:
Nit:

```suggestion
    let mut rows = Vec::new();
```

... makes it explicit what data structure we are creating here, and is consistent with how we initialize `results` above.